### PR TITLE
Features/django version update

### LINKED ISSUE #74: Upgrade Django
Upgrade django to the latest version of 4.2 and python 3.12

### PR DESCRIPTION
**Pull request by aadilrashidd**
resolves #74

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -6,6 +6,6 @@ django-filter
 django-flat-responsive
 django-flat-theme
 django
```
Comment (ZeroCoolHacker) [inline requirements.txt:8]: lets use django 4.2

File: requirements.txt [hunk 2]
```diff
@@ -6,6 +6,6 @@ django-filter
 django-flat-responsive
```
Comment (ZeroCoolHacker) [inline requirements.txt:6]: https://pypi.org/project/django-flat-responsive/

File: requirements.txt [hunk 3]
```diff
@@ -6,6 +6,6 @@ django-filter
...
 django-flat-theme
```
Comment (ZeroCoolHacker) [inline requirements.txt:7]: https://pypi.org/project/django-flat-theme/

**Review by ZeroCoolHacker — CHANGES_REQUESTED** [review]

**Comment by aadilrashidd** [comment]
everything is up to date .

**Pull request merged**